MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -5663,12 +5663,17 @@ public TargetRoll getTargetForAcquisition(IAcquisitionWork acquisition,
                 et = ((MissingEquipmentPart) acquisition).getType();
             }
 
+            StringBuilder partAvailabilityLog = new StringBuilder();
+            partAvailabilityLog.append("Part Rating Level: " + partAvailability);
+            partAvailabilityLog.append("(" + EquipmentType.ratingNames[partAvailability] + ")");
```

Comment:
Why not just chain appends?

Reply:
I found it easier to read the code that way, but I can probably just put the append on the next line as well.

Reply:
I'd recommend the next line append, as it is somewhat cleaner in file.